### PR TITLE
commandbox 3.4.0

### PR DESCRIPTION
**Pull request by bdw429s**

### CODE REVIEW

File: Formula/commandbox.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def install
   end
 
   test do
-    system "box", "--commandbox_home=~/", "version"
-    system "box", "--commandbox_home=~/", "help"
+    system "/usr/local/bin/box", "--commandbox_home=~/", "version"
```
Comment (ilovezfs) [inline Formula/commandbox.rb]: @MikeMcQuaid amazingly this passed audit :see_no_evil:

Comment (ilovezfs) [inline Formula/commandbox.rb]: probably

File: Formula/commandbox.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@ def install
...
+    system "/usr/local/bin/box", "--commandbox_home=~/", "help"
```
Comment (bdw429s) [inline Formula/commandbox.rb]: Sorry guys, I misinterpreted the message from the Jenkins build that I needed to put in the actual path to the binary.  I didn't realize that I actually needed the text `#{bin}`  I'll update now.

**Comment by bdw429s** [comment]
@ilovezfs Ok updated the path.  We'll see if the audit is still happy.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**